test/contracts/storage/atlas_stake_store.js

```diff
@@ -209,6 +209,10 @@ describe('AtlasStakeStore Contract', () => {
       await expect(releaseStake(shelterer, otherAddress)).to.be.eventually.rejected;
     });
 
+    it('can not release a stake if refund address equal 0x0', async () => {
+      await expect(releaseStake(shelterer, '0x0')).to.be.eventually.rejected;
```

Comment:
I'm not actually sure if it fails because address is 0 or because it does not have correct format (40 digits)

Reply:
++. I think you should check against 0x0......0 instead:D